extensions/AzureCosmosDBMongoDB/AzureCosmosDBMongoDB.TestApplication/appsettings.json

```diff
@@ -0,0 +1,68 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Trace",
+            "Microsoft.AspNetCore": "Trace"
+        }
+    },
+    "UseRealEmbeddingGenerator": false,
+    "KernelMemory": {
+        "Services": {
+            "AzureCosmosDBMongoDB": {
+                // For other options see <AzureAISearchConfig>.
+                "ConnectionString": "mongodb+srv://akataria3011:Basket24ball@akatarialangchaintesting.mongocluster.cosmos.azure.com/?tls=true&authMechanism=SCRAM-SHA-256&retrywrites=false&maxIdleTimeMS=120000",
```

Comment:
@aayush3011 looks like you checked in some personal credentials. Please invalidate the deployment ASAP thanks

Reply:
Removed

Reply:
@aayush3011 double checking, did you also delete the resource in the portal? 

(the secret, even if removed from the PR, is still visible in git history)